lib/swagcov/coverage.rb

```diff
@@ -27,14 +28,12 @@ def report
         # https://github.com/rails/rails/tree/main/actionmailbox/app/controllers/action_mailbox
         next if path.include?("/active_storage/") || path.include?("/action_mailbox/")
 
-        if ignore_path?(path)
+        if dotfile.ignore_path?(path)
           @ignored += 1
           @routes_ignored << { verb: route.verb, path: path, status: "ignored" }
           next
         end
 
-        next if only_path_mismatch?(path)
-
```

Comment:
I think we should keep this or provide a flag/argument (however it works for rake tasks) to report the same way as ignored or not. To me, if I specify I `only` care about `^v2` I don't care about `^v1` therefore remove the cruft from the printout/don't count it against the total ignored since it's not explicitly ignored. If you really want to see it, perhaps we mark it as `skipped` instead.

Reply:
`ignore` is intended to behave similar to rubocops ignore as in it is a todo.

Reply:
Done. Makes sense